lambdas/evaluate-gpg45-scores/src/main/java/uk/gov/di/ipv/core/evaluategpg45scores/EvaluateGpg45ScoresHandler.java

```diff
@@ -205,6 +207,23 @@ public APIGatewayProxyResponseEvent handleRequest(
         }
     }
 
+    private void checkCorrelation(String userId, List<VcStatusDto> currentVcStatuses)
+            throws HttpResponseExceptionWithErrorBody {
+        if (!userIdentityService.checkNameAndFamilyNameCorrelationInCredentials(
+                userId, currentVcStatuses)) {
+            LOGGER.error("Failed to correlate gathered names for userId:{}", userId);
```

Comment:
Does this need to generate an audit message as well? ("Add appropriate logging and audit messages.")